mirgecom/profiling.py

```diff
@@ -110,6 +110,7 @@ def array_kernel_exec_hook(self, knl, queue, gs, ls, *actual_args, wait_for):
         evt = knl(queue, gs, ls, *actual_args, wait_for=wait_for)
 
         name = knl.function_name
+        knl.name = knl.function_name  # "name" is used by the LoopKernels
```

Comment:
Not a fan of modifying `cl.Kernel` objects. IMO this is not an improvement.

Reply:
I undid this.